### DISCUSSION TITLE
Global: Last touched "Only work on focused FX"

### DISCUSSION THREAD

**Discussion by alucast** [helgoboss/helgobox#1325] [Ideas]
Is this possible? Because it's cool to have control of last touched but sometimes after a while I forget it's linked to some parameter and I change settings, maybe a way to limit that to only work when the FX is focused or something similar would be cool.

**Comment by helgoboss** [comment]
No. I guess it would make sense as an additional FX parameter selector "Last touched" of the "FX parameter: Set value" target.

**Comment by alucast** [comment]
Yes, that could probably help, thank you!

**Comment by helgoboss** [comment]
In that case, please create a feature request.

**Comment by helgoboss** [comment]
Ah wait, it's in category "ideas" already, no need to create another one ;) Sorry.
